tools.js

```diff
@@ -59,6 +59,9 @@ module.exports = (function () {
 		timid: {name:"Timid", plus:'spe', minus:'atk'}
 	};
 
+	var init = typeof global.Symbol === 'function' ? Symbol('init') : '_init';
```

Comment:
I'm not sure why they need to be protected at all. Why `_ownInit`?

Maybe it should just be `initMod` and `inheritMod`.